### ISSUE TITLE
EEP26RockerSwitch2RockerAction .AI .AO .BI .BO values

### ISSUE THREAD

**Issue by winnall** [dog-gateway/enj-library#5]
I'm trying to use the EnJ Library with a PTM 210 DB, which is a F6-02-01 rocker switch. I use the following code to check the values I receive via an ESP3 packet from the switch:

    private void printRockerAction(F60201 rockerSwitch, int channel) {
        EEP26RockerSwitch2RockerAction rockerSwitchAction
                            = (EEP26RockerSwitch2RockerAction) rockerSwitch.getChannelAttribute(
                                    channel, EEP26RockerSwitch2RockerAction.NAME);
        Boolean[] values = rockerSwitchAction.getValue();
        System.out.println("  Channel " + channel);
        System.out.println("    [AI] " + values[EEP26RockerSwitch2RockerAction.AI]);
        System.out.println("    [AO] " + values[EEP26RockerSwitch2RockerAction.AO]);
        System.out.println("    [BI] " + values[EEP26RockerSwitch2RockerAction.BI]);
        System.out.println("    [BO] " + values[EEP26RockerSwitch2RockerAction.BO]);
     }

If I hold down the AO button on the switch and press the energy bow, the above code writes for F60201.CHANNEL_1

    [AI] true

And when I hold down the AI button it outputs

    [AO] true

The same happens with BO and BI. Is it possible that

    public class EEP26RockerSwitch2RockerAction extends EEPAttribute<Boolean[]>
    {
        public static int AI = 0;
        public static int AO = 1;
        public static int BI = 2;
        public static int BO = 3;

should be

    public class EEP26RockerSwitch2RockerAction extends EEPAttribute<Boolean[]>
    {
        public static int AI = 1;
        public static int AO = 0;
        public static int BI = 3;
        public static int BO = 2;

?

Steve

**Comment by winnall** [comment]
I tried changing the values of of `AI` etc in `EEP26RockerSwitch2RockerAction ` as above, but I still get the same result.

**Comment by dbonino** [comment]
Dear Steve, 
I have checked the specification and the EnJ code and you are right. There was a position switch in the code "decoding" the RPS message payload. The right point was in the `private boolean[] decodeAction(byte action)` method of the `F6020102RockerSwitchMessage`.

Thank you for pointing out the error. I've just committed an updated version which should fix this bug. If you want to give it a try...
